pkg/runtime/framework/plugins/mpi/mpi.go

```diff
@@ -154,50 +161,64 @@ func (m *MPI) EnforceMLPolicy(info *runtime.Info, trainJob *trainer.TrainJob) er
 				}...,
 			)
 		}
-		for cIdx, container := range ps.Containers {
-			if container.Name != constants.JobLauncher && container.Name != constants.JobTrainerNode {
-				continue
-			}
+		containers, err := enforceMPIParametersToContainers(info, ps.Name, psIdx, info.TemplateSpec.PodSets[psIdx].Containers...)
+		if err != nil {
+			return err
+		}
+		info.TemplateSpec.PodSets[psIdx].Containers = containers
+		initContainers, err := enforceMPIParametersToContainers(info, ps.Name, psIdx, info.TemplateSpec.PodSets[psIdx].InitContainers...)
```

Comment:
Why do we need to enforce MPI params to the InitContainers ?

Reply:
However, I decided to remove all initContainer impls from this PR since we do not need to launch ssh server for Launcher.

Reply:
Done.